src/test/java/org/kainos/ea/service/JobRoleServiceTest.java

```diff
@@ -0,0 +1,50 @@
+package org.kainos.ea.service;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.kainos.ea.dao.JobRoleDao;
+import org.kainos.ea.exception.FailedToGetJobRolesException;
+import org.kainos.ea.model.JobRole;
+import org.kainos.ea.util.DatabaseConnector;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+@ExtendWith(MockitoExtension.class)
+public class JobRoleServiceTest {
+
+    JobRoleDao jobRoleDao = Mockito.mock(JobRoleDao.class);
+    DatabaseConnector databaseConnector = Mockito.mock(DatabaseConnector.class);
+
+    JobRoleService jobRoleService = new JobRoleService(jobRoleDao, databaseConnector);
+
+    Connection conn;
+
+    @Test
```

Comment:
add a unit test for the databaseConnector.getConnection() method throwing an exception

Reply:
this still needs added